app/src/main/java/com/vitorpamplona/amethyst/ui/navigation/Routes.kt

```diff
@@ -35,17 +35,32 @@ sealed class Route(
     val arguments: List<NamedNavArgument> = emptyList(),
     val buildScreen: (AccountViewModel, AccountStateViewModel, NavController) -> @Composable (NavBackStackEntry) -> Unit
 ) {
+    val base: String
+        get() = route.substringBefore("?")
```

Comment:
It turns out we sometimes use the route without any optional params added (e.g. for comparing whether or not we are on the same route), so we can use this `base` delegated property whenever we need to check that.